v4/integrations/nrawssdk-v1/nrawssdk_test.go

```diff
@@ -158,14 +158,14 @@ var (
 		{Name: "External/lambda.us-west-2.amazonaws.com/http/POST", Scope: "OtherTransaction/Go/" + txnName, Forced: false, Data: nil},
 	}, txnMetrics...)
 	datastoreMetrics = append([]internal.WantMetric{
-		{Name: "Datastore/DynamoDB/all", Scope: "", Forced: true, Data: nil},
-		{Name: "Datastore/DynamoDB/allOther", Scope: "", Forced: true, Data: nil},
+		{Name: "Datastore/dynamodb/all", Scope: "", Forced: true, Data: nil},
+		{Name: "Datastore/dynamodb/allOther", Scope: "", Forced: true, Data: nil},
 		{Name: "Datastore/all", Scope: "", Forced: true, Data: nil},
 		{Name: "Datastore/allOther", Scope: "", Forced: true, Data: nil},
-		{Name: "Datastore/instance/DynamoDB/dynamodb.us-west-2.amazonaws.com/unknown", Scope: "", Forced: false, Data: nil},
-		{Name: "Datastore/operation/DynamoDB/DescribeTable", Scope: "", Forced: false, Data: nil},
-		{Name: "Datastore/statement/DynamoDB/thebesttable/DescribeTable", Scope: "", Forced: false, Data: nil},
-		{Name: "Datastore/statement/DynamoDB/thebesttable/DescribeTable", Scope: "OtherTransaction/Go/" + txnName, Forced: false, Data: nil},
+		{Name: "Datastore/instance/dynamodb/dynamodb.us-west-2.amazonaws.com/unknown", Scope: "", Forced: false, Data: nil},
+		{Name: "Datastore/operation/dynamodb/DescribeTable", Scope: "", Forced: false, Data: nil},
+		{Name: "Datastore/statement/dynamodb/thebesttable/DescribeTable", Scope: "", Forced: false, Data: nil},
+		{Name: "Datastore/statement/dynamodb/thebesttable/DescribeTable", Scope: "OtherTransaction/Go/" + txnName, Forced: false, Data: nil},
```

Comment:
`dynamodb` is now lower cased in all the tests because the otel spec wants it that way.

Reply:
For these type of name changes, if they affect anything like alerts or the UI, we should make sure to include that high up in the changelog for this shim.

Reply:
Since we're not including any metrics in this round of the shim, these changes won't have any specific effect on Alerts. Attributes on spans will definitely be changing, so that's def something we should call out.